app/views/signup/start.html.erb

```diff
@@ -10,11 +22,24 @@
                                 context: self,
                                 errors: @handler_result.try(:errors)) %>
     <%# TODO put select in form helper %>
-    <%# TODO put select options in translations %>
-    <%= f.select :role, options_for_select(%w(Instructor Student).map{|role| [role, role]},
-                        [saved_role].compact) %>
-    <%= fh.text_field name: :email, value: saved_email %>
-    <%= f.submit (t :".next"), class: 'btn btn-primary' %>
+
+    <%= f.select :role, options_for_select(signup_roles.invert.to_a, saved_role) %>
```

Comment:
I think `saved_role` might be capitalized... so the selected option might not work?  Or rather, might it be working now because the english translation has it capitalized...